src/_data/sites/binnyva.json

```diff
@@ -0,0 +1,8 @@
+{
+    "url": "https://binnyva.com/",
+    "name": "Binny V A",
+    "description": "Personal website of Binny V A - Philosopher programmer who specializes in web development and stoicism.",
+    "twitter": "binnyva",
+    "authoredBy": ["binnyva"],
+    "source_url": "https://github.com/binnyva/Site_binnyva.com"
```

Comment:
Is this a private repo? It seems to 404 for me.

Reply:
Sorry - missed that. Made it public now. Thanks :-)